packages/ember-data/lib/system/store.js

```diff
@@ -185,6 +185,7 @@ Store = Ember.Object.extend({
       store: this
     });
     this._pendingSave = [];
+    this._containerCache = Ember.create(null);
```

Comment:
It's where the store holds its Serializers and Adapters instances - since each is now one instance per store rather than a singleton.